### ISSUE TITLE
New Gateway Rinkeby - Feedback & Issues 

### ISSUE THREAD

**Issue by MitchellUnknown** [omgnetwork/optimism-v2#418]
**Gateway**
1. Add a "Copy address" & "Blockexplorer" button next to the Network switcher on the top right corner

![Copy and Explorer button](https://user-images.githubusercontent.com/90399189/155255768-4cfbee61-fd05-4b08-9599-d8507b54113f.png)

2. You could make a clearer distinction between the transfer and bridge buttons. For example, a color change. I think it will give the user a better understanding for the meaning of the buttons. 

![Transfer Button](https://user-images.githubusercontent.com/90399189/155255794-ffe26c2e-4443-4882-8a6b-ee9ce6f2033f.png)

3. Make the "browser tab icon" at the top of the gateway and explorer more in line with the new design.

![Tab Icon Gateway](https://user-images.githubusercontent.com/90399189/155255837-ed6ba9e7-5144-4dac-859f-634f96c73bea.png)

4. On the pc version the information on the bottom is a bit off. I tried to line it up in the left corner. Also it could be a good idea to add some more core links/information: testnet, boba docs, boba website, forum

![Information Bottom](https://user-images.githubusercontent.com/90399189/155255863-e27f16cf-7978-4e46-8e76-4d8de07d2d33.png)

5. This is a personal preference, could be wrong. But the Boba Logo 2 feels out of place for some reason on the gateway at the pc. It feels a bit big. And I feel it's better on the bottom of the page. Leave the top clean.

![Logo Boba](https://user-images.githubusercontent.com/90399189/155255921-5e98177c-0d07-4647-ad99-04a690a43722.png)

6. Maybe dim the icon of the network you are not on. So give the user extra feedback on which network they are currently on.

![image](https://user-images.githubusercontent.com/90399189/155302416-eafcd534-dfa1-4f0a-9ea6-c2afbbb4e890.png)

7. Some of the interaction pop up still has the old blue design theme. Replace them with the new design theme, like the DAO pop up.

![Window Pop Up](https://user-images.githubusercontent.com/90399189/155298573-378e860d-141d-4a3b-9d35-30084124dc20.png)

8. When I am using the gateway, I always want to know how many steps I need to take to complete my interaction. Some users don't understand you have to approve blind signing first, to interact with some of the blockchain interactions. If you help them by showing the steps in advance, this will make their life easier. 

![image](https://user-images.githubusercontent.com/90399189/155301797-1f72b010-ad90-4d9a-a405-3edd16ee98b6.png)

**Explorer**
1. Issue with the boba explorer, blocks will show up all weird when a new tx is added to the page.

![Explorer Bug New Block or TX](https://user-images.githubusercontent.com/90399189/155255986-8eedaa8d-5ea5-463d-936e-df186b048721.png)

2. The txn hash and wallet addresses need to be 'shortened' on the explorer, otherwise you have to scroll to the side, to see the most important information from the tx. 

![Hash is to long](https://user-images.githubusercontent.com/90399189/155256002-8f0b2cc9-0614-4a51-8f81-0787b0cc514d.png)

3. Suggestion to give the Boba explorer a small "Boba Vibe" make over.

![Boba Explorer Design (More Boba)](https://user-images.githubusercontent.com/90399189/155256021-1f6847de-f3a3-4192-9073-ee8f5f89099e.png)

4. The mobile Boba Blockexplorer need some adjustments, see Etherscan mobile version. Some parts are weird on mobile, for me atleast.

![Explorer Mobile version](https://user-images.githubusercontent.com/90399189/155256038-ff2d3b65-8bdd-4e1b-a071-7283db3c0af7.png)

**Comment by CAPtheorem** [comment]
This is great feedback - will try to incorporate as much as we can over the next few days. FYI @sk-enya.

**Comment by sk-enya** [comment]
Add fix for the few suggestion from this thread.
https://github.com/omgnetwork/optimism-v2/pull/434

**Comment by sk-enya** [comment]
1. added copy button to copy address
2. button styling for transfer and bridge as outlined or contained
3. favicon change for browser tab indicator.
4. added boba docs, boba website and other placeholder in page footer
5. Reduced the size of logo in case of PC
6. Darken the color of icon base on current chain

**Comment by CAPtheorem** [comment]
Yes thanks @sk-enya much progress. Still a ways to go - @MitchellUnknown thanks for your extremely detailed writeup with many more excellent suggestions

**Comment by MitchellUnknown** [comment]
> Yes thanks @sk-enya much progress. Still a ways to go - @MitchellUnknown thanks for your extremely detailed writeup with many more excellent suggestions

❤️ No problem, great work guys!
